CodeGeneration/Package.swift

```diff
@@ -1,24 +1,17 @@
-// swift-tools-version:5.3
-// The swift-tools-version declares the minimum version of Swift required to build this package.
+// swift-tools-version:5.5
 
 import PackageDescription
 
 let package = Package(
     name: "Codegen",
     products: [
-        // Products define the executables and libraries a package produces, and make them visible to other packages.
```

Comment:
I can see why Apple would add these comments to the template they generate. But they are just noise in an established codebase.